ee/server/service/scripts.go

```diff
@@ -383,3 +376,67 @@ func (svc *Service) GetHostScriptDetails(ctx context.Context, hostID uint, opt f
 
 	return svc.ds.GetHostScriptDetails(ctx, h.ID, h.TeamID, opt)
 }
+
+func (svc *Service) BatchSetScripts(ctx context.Context, maybeTmID *uint, maybeTmName *string, payloads []fleet.ScriptPayload, dryRun bool) error {
```

Comment:
Should we add the usual auth tests for the new endpoint?

Reply:
I tried, but I quickly realized why none of the `ee/` service methods has them! there's an import cycle we need to solve in order to use the logic that sets ups the test suite that makes this test possible. 😞